app/serializers/api/v1/message_serializer.rb

```diff
@@ -3,18 +3,18 @@ class MessageSerializer < ApplicationSerializer
     embed :ids, include: true
 
     attributes :id, :body, :state, :is_private, :created_at,
-      :updated_at, :offer_id, :item_id, :designation_id, :order_id
+      :updated_at, :messageable_type, :messageable_id, :lookup
```

Comment:
As per @patrixr note re backwards compatiblity, can you add item_id, order_id and message_id methods here.

E.g.

```
def item_id
  object.messageable_type == 'Item' ? object.messageable_id if  : nil
end

def item_id__sql
  "CASE WHEN messageable_type='Item' THEN messageable_id ELSE NULL END"
end

```

Reply:
@patrixr @steveyken 
cool. I had added this in one of my previous commit, and again removed it thinking it wont be require. I'll add it back. Thanks